stats/stats_test.go

```diff
@@ -380,6 +380,49 @@ func TestHistogramNegativeNumbers(t *testing.T) {
 	}
 }
 
+func TestTransferHistogramWithDifferentScales(t *testing.T) {
+	tP := []float64{75.}
+	var b bytes.Buffer
+	w := bufio.NewWriter(&b)
+	h1 := NewHistogram(2, 15)
+	h1.Record(30)
+	h1.Record(40)
+	h1.Record(50)
+	h2 := NewHistogram(0, 10)
+	h2.Record(20)
```

Comment:
add another point in same bucket for h2 so you see the bug that you only added 1 count (do it before fixing the bug so you know you have a test before the fix)